README.Rmd

```diff
@@ -60,7 +60,13 @@ library(austraits)
 
 In our website, we have also included information about the [structure of the database](https://traitecoevo.github.io/austraits/articles/structure.html), [definitions of the traits](https://traitecoevo.github.io/austraits/articles/dictionary.html) and some [tutorials](https://traitecoevo.github.io/austraits/articles/austraits.html) that uses our functions as well as some `tidyverse` functions to create some commonly used data output formats. We highly recommend starting here before jumping into the database! 
 
+If you prefer to view the vignettes in R, we recommend installing the package with `build_vignettes = TRUE`. 
```

Comment:
Should we mention something here about lightweight install, need all suggest packages for vignettes?

Reply:
Yes good idea! 
I might i'll at the beginning:

For access to the full capabilities of austraits including plotting functions and those required in the vignettes, use: 
```#install.packages("remotes")
remotes::install_github("traitecoevo/austraits", dependencies = TRUE, upgrade = "ask")```

Otherwise, for a lightweight installation where dependencies for plotting and the vignettes will not be installed, use:
`remotes::install_github("traitecoevo/austraits", upgrade = "ask")`

Thoughts